tap_mssql/connection.py

```diff
@@ -50,12 +50,12 @@ def __init__(self, config):
             if "Adaptive Server is unavailable or does not exist" in message:
                 if "timed out" in message:
                     raise SymonException('Timed out connecting to database. Please ensure all the form values are correct.', 'odbc.ConnectionTimeout')
-                raise SymonException(f'The host "{config["host"]}" was not found. Please check the host name and try again.', 'odbc.HostNotFound')
+                raise SymonException(f'Sorry, we couldn\'t connect to the host "{config["host"]}". Please ensure all the form values are correct.', 'odbc.ConnectionFailed')
             raise
         except pymssql._mssql.MSSQLDriverException as e:
-            # using invalid url like 'asdas' for host gives this error
+            # invalid host url like 'asd' raises this error, but it might not be the only reason for this error
             if "Connection to the database failed for an unknown reason" in str(e):
-                raise SymonException(f'The host "{config["host"]}" was not found. Please check the host name and try again.', 'odbc.HostNotFound')
+                raise SymonException(f'Sorry, we couldn\'t connect to the host "{config["host"]}". Please ensure all the form values are correct.', 'odbc.ConnectionFailed')
```

Comment:
For host not found, pymysql raises this error, but this error doesn't indicate it is due to invalid host only (Could be some configuration error on mysql side).